src/lib/utils/mobile.ts

```diff
@@ -0,0 +1,3 @@
+export const isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini|Windows Phone/i.test(
```

Comment:
just so I understand, in what cases would this be used instead of `useSmallScreen`?

Reply:
`useSmallScreen` responds to screen size, so I named it that way to correctly reflect that. As in, it doesn't matter if you're on desktop or mobile, if you have a small screen it'll be true.

In some instances, we actually care if you're on mobile or not, like when determining padding that behaves differently on mobile devices. Might not be many cases because we mostly care about responsiveness, but I did need it for the case I used it in